--- src/autenticacao.gov/pteid-mw-pt/_src/eidmw/dialogs/dialogsQT/dlgs.cpp	2020-04-08 17:20:23.939513211 +0200
+++ src/autenticacao.gov/pteid-mw-pt/_src/eidmw/dialogs/dialogsQT/dlgs_fixed.cpp	2020-04-08 17:00:06.599489218 +0200
@@ -434,7 +434,7 @@
   char csCommand[150];
   Type_WndGeometry *pWndGeometry = (Type_WndGeometry *)wndGeometry;
 
-  sprintf(csCommand,"%s/%s %i %s", "/usr/local/bin", csServerName.c_str(),index,csFilename );
+  sprintf(csCommand,"%s/%s %i %s", "/usr/bin", csServerName.c_str(),index,csFilename );
 
   if  ( ( pWndGeometry != NULL )
         && ( pWndGeometry->x >= 0 ) && ( pWndGeometry->y >= 0 )
@@ -482,4 +482,4 @@
     if ( outWndGeometry->y > desktop_height ) return false;
 
     return true;
-}
\ No hay ningún carácter de nueva línea al final del fichero
+}
